src/frame-writer.cpp

```diff
@@ -170,6 +170,9 @@ void FrameWriter::init_video_stream()
     videoCodecCtx->height = params.height;
     videoCodecCtx->time_base = (AVRational){ 1, FPS };
 
+    if (params.force_yuv || params.opencl)
```

Comment:
I would like to see a comment explaining why this is necessary (or what it is a workaround for)

Reply:
This is for drivers like radeonsi that do not support b-frames. Without this, attempting to use opencl will throw an error message that b-frames are not supported. I'm thinking we might istead add a `-b` option the user can pass that will be fed to `max_b_frames` and just not set it at all if `-b` is not passed. What do you think?